spectrum-offchain-lm/src/executor.rs

```diff
@@ -300,6 +311,54 @@ where
                                         NodeSubmitTxError::DoubleSpend => {
                                             self.backlog.lock().await.suspend(ord).await;
                                         }
+
+                                        NodeSubmitTxError::ScriptsOfInputsDontPassVerification => {
```

Comment:
This is not a business error. In fact, if scripts don't pass verification it's a result of a bug in bots impl.